ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitSendDetailsWidget.kt

```diff
@@ -98,12 +98,9 @@ class UiKitSendDetailsWidget @JvmOverloads constructor(
     }
 
     fun showFeeLoading(isLoading: Boolean) {
-        binding.progressBarFees.isVisible = isLoading
-        if (binding.textViewFee.text.isNullOrBlank()) {
-            binding.imageViewFeesInfo.isVisible = false
-        } else {
-            binding.imageViewFeesInfo.isInvisible = isLoading
-        }
+        val canBeVisible = binding.textViewFee.isVisible
+        binding.progressBarFees.isVisible = canBeVisible && isLoading
+        binding.imageViewFeesInfo.isVisible = canBeVisible && !isLoading
```

Comment:
this is very hacky

Reply:
better then before
it is just an a condition not to show view when another view is GONE
because we have progress and info view
and text
progress show shown when we load fee
info when not loading
text is a main view if we do not have text of fees we can't show progress or info

another fix can be done in layout
wrap this views in Linear layout and hide it all

Reply:
The linear layout sounds good to me